client/components/room/Video.jsx

```diff
@@ -53,32 +52,31 @@ class Video extends React.Component {
     this.onSetSessionDescriptionError = this.onSetSessionDescriptionError.bind(this);
     this.handleRemoteStreamAdded = this.handleRemoteStreamAdded.bind(this);
     this.handleRemoteStreamRemoved = this.handleRemoteStreamRemoved.bind(this);
-    this.hangup = this.hangup.bind(this);
     this.handleRemoteHangup = this.handleRemoteHangup.bind(this);
     this.stop = this.stop.bind(this);
-    this.requestTurn = this.requestTurn.bind(this);
     this.toggleVideo = this.toggleVideo.bind(this);
     this.toggleAudio = this.toggleAudio.bind(this);
     this.handleCopy = this.handleCopy.bind(this);
     this.sendToMessenger = this.sendToMessenger.bind(this);
+    this.onUnload = this.onUnload.bind(this);
   }
 
   componentDidMount() {
-    let localVideo = document.getElementById('localVideo');
-    let remoteVideo = document.getElementById('remoteVideo');
-    if (location.hostname !== 'localhost') {
-      this.requestTurn(
-        'http://numb.viagenie.ca?username=andy.yeo@gmail.com&key=hackreactor'
-      );
-    }
+    const localVideo = document.getElementById('localVideo');
```

Comment:
I'm confused how gotStream() has access to these variables considering that const and let have block scope. Shouldn't they be inaccessible to the gotStream(), or is there something I'm missing?